README.md

```diff
@@ -25,6 +29,21 @@ like `bug_12345_my_cool_experiment.toml`.
 
 Lines starting with a `#` are comments and have no effect.
 
+### Outcome snippets
+
+Outcome snippets, which define a collection of summaries with a common theme (e.g. "performace", "Picture in Picture use"),
+are stored in the `outcomes/` directory and file names serve as unique identifiers.
+
+Configuration files have a set of [`[metrics]` definitions](#defining-metrics) and [`[data_sources]`](#defining-data-sources).
+A `friendly_name` and `description` are required at the top of the outcome snippet config file.
```

Comment:
Do we need to add `applications` here to specify what applications an outcome snippet supports?

See https://github.com/mozilla/jetstream/pull/474#discussion_r569642670

```suggestion
A `friendly_name` and `description` are required at the top of the outcome snippet config file.

For specifying applications an outcome snippet computes metrics and statistics for, `application` is required to specify a list of supported `app_id`s. For example: `applications = ['firefox-desktop']` or `applications = ['org.mozilla.fenix', 'org.mozilla.fenix.nightly', 'org.mozilla.fennec.aurora']`
```

Reply:
I think we should be consistent about using app_id or app_name, but yes, I think we need to represent application somehow.

It might be useful to organize them in separate paths, e.g. fenix/my_outcome.toml or firefox_desktop/my_outcome.toml.

Reply:
Is it possible that some outcomes could target both Fenix and desktop? E.g. outcomes related to Gecko? (Or Fenix and iOS, who knows what the Nimbus future will look like ;)) So this might not be compatible with having outcome definitions in separate paths>

Reply:
I'm okay with duplicating the outcomes in that case. Right now, there's no chance of being able to reuse the definitions because the table structures are different. In some post-Glean world, that might look different, but the feature sets of the apps are different enough that I don't think we'll regret the duplication too deeply.